server/handlers.go

```diff
@@ -817,8 +845,13 @@ func (s *Server) getHandler(w http.ResponseWriter, r *http.Request) {
 
 	w.Header().Set("Content-Type", contentType)
 	w.Header().Set("Content-Length", strconv.FormatUint(contentLength, 10))
-	w.Header().Set("Content-Disposition", fmt.Sprintf("attachment; filename=\"%s\"", filename))
-	w.Header().Set("Connection", "close")
+	w.Header().Set("Connection", "keep-alive")
+
+	if action == "inline" {
```

Comment:
a little redundant. why not setting a `disposition` variable having either `inline` or `attachment` as value and format it in a single `fmt.Sprintf("%s; filename=\"%s\"", disposition, filename)`?